xbmc/video/dialogs/GUIDialogVideoInfo.h

```diff
@@ -31,18 +31,18 @@ class CGUIDialogVideoInfo :
 public:
   CGUIDialogVideoInfo(void);
   virtual ~CGUIDialogVideoInfo(void);
-  virtual bool OnMessage(CGUIMessage& message);
-  virtual bool OnAction(const CAction &action);
+  bool OnMessage(CGUIMessage& message) override;
+  bool OnAction(const CAction &action) override;
```

Comment:
i prefer to leave the `virtual` keywords because it's easier to spot a keyword at the beginning of a line as opposed to the end, but to each his own